src/qtxdg/xdgmimeappsglibbackend.cpp

```diff
@@ -42,6 +42,9 @@ static QList<XdgDesktopFile *> GAppInfoGListToXdgDesktopQList(GList *list)
                 if (df->load(file) && df->isValid()) {
                     dl.append(df);
                 }
+                else {
```

Comment:
Coding style not followed.

Reply:
> Coding style not followed.

`} else {` or something else? I did my best to follow the style ;)

Reply:
```} else {```

there isn't any "example" in the all file.
Then merge it.

Reply:
Will change and merge it. Thanks!